config/chainbridge-runtime-config.cere.dev.js

```diff
@@ -28,8 +28,8 @@ window.__RUNTIME_CONFIG__ = {
         tokens: [
           {
             address: "0xcD54b7066839B25A0E5fDf76866D338cc7257179",
-            name: cereTokenName,
-            symbol: cereTokenSymbol,
+            name: "CN",
+            symbol: "CS",
```

Comment:
Can we use more meaningful names here?

Reply:
This is the name and symbol from contract `0xcD54b7066839B25A0E5fDf76866D338cc7257179`
https://mumbai.polygonscan.com/address/0xcD54b7066839B25A0E5fDf76866D338cc7257179
[CN (CS)](https://mumbai.polygonscan.com/token/0xcD54b7066839B25A0E5fDf76866D338cc7257179)
<img width="349" alt="Screenshot 2023-06-30 at 15 27 15" src="https://github.com/Cerebellum-Network/chainbridge-ui/assets/32759781/3c0bb60b-26d1-41cc-8f65-df127971f69b">